decoder/block_candidates.go

```diff
@@ -9,7 +9,7 @@ import (
 	"github.com/hashicorp/hcl/v2"
 )
 
-func blockSchemaToCandidate(blockType string, block *schema.BlockSchema, rng hcl.Range) lang.Candidate {
+func blockSchemaToCandidate(blockType string, block *schema.BlockSchema, rng hcl.Range, prefillRequiredAttributes bool) lang.Candidate {
```

Comment:
Nitpick: we might benefit from turning this function into a method on the `Decoder` as that would avoid the need for that extra argument.

Reply:
Made it into a method on `Decoder`, but didn't move the method to `decoder.go` because otherwise there is nothing left in this file